lib/google/gax/version.rb

```diff
@@ -29,6 +29,6 @@
 
 module Google
   module Gax
-    VERSION = '0.5.1'.freeze
+    VERSION = '0.5.2'.freeze
```

Comment:
This type of the change would require an increment on the minor version part, not the patch part (see http://semver.org/).